### ISSUE TITLE
[BUG] evennia command hangs if server start fails

### ISSUE THREAD

**Issue by TehomCD** [evennia/evennia#2057]
#### Describe the bug
If you try to issue `evennia start` and the server has an error, the application will hang.

#### To Reproduce
1. Add raising a ValueError to the webserver's addService, or some other non-import-time error.
2. attempt to start the game with `evennia start`
3. See `Server starting ...` and the error in server.log
4. Freeze forever.

#### Expected behavior
The CLI to display the error to the user and return to the commandline.

#### Environment, Evennia version, OS etc
linux, latest rev

#### Additional context
It occurred to me that this probably wasn't intended behavior as I was trying to investigate adding a retry mechanism on failures.

**Comment by Griatch** [comment]
@TehomCD Interestingly, when adding `raise ValueError("TEST")` in `WSGIWebServer.startService`, I do get the frozen startup as you say, but if I Ctrl-C and then try to start again I also get the `Address already in use` error on port 4005 you reported in #2028. Maybe the various webserver issues reported (#1493, #1758, #2053) could be connected to this too in that case - the webserver dying in various ways could be causing weird side effects for the server as a whole.

**Comment by TehomCD** [comment]
Oh that's a great discovery, I hadn't checked about the webserver port still being occupied. I think you're right that these all seem related.

**Comment by Griatch** [comment]
@TehomCD I have made sure to handle errors in the webserver startup. The result is that the Webserver will cleanly fail to start (and log this fact) but the rest of the server will start normally. So no more lockups and blocked ports. It's possible that there are other error sources that still can lock things up, but this one seems to be the most likely one. Maybe it will also help with other lock-up situations. 

Implemented in `develop` branch; so closing this for book keeping.
